### ISSUE TITLE
Single usage highlight

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1537]
It should works as an extension of the "one-time use variables", but this last one will not warn when you have an intermediare statement (which is fine). My suggestion is just notify user with a weak warning when it happen.

**Currently:**

```php
$a = 1;  // Warning: one-time use variable.
$b = $a;
```

```php
$a = 1;  // No warning.
$b = 2;
$c = $a;
```

**After:**

```php
$a = 1;  // Warning: one-time use variable.
$b = $a;
```

```php
$a = 1;  // Weak warning: one-time use variable.
$b = 2;
$c = $a;
```

It can be useful to help us to move variables closest to the usage too and additionally inline it easy.

**Comment by kalessil** [comment]
Looks good, need to evaluate performance impact.

**Comment by imme-emosol** [comment]
To me, declaring a variable immediately above it's one-time usage is something that is done deliberately.., for readability or something like that. The same might be true for one-time used variables that are declared at the top of a method/function definition.
All other cases of one-time used variables seem style-issues to me, for instance as remnants of refactoring.

To my surprise this inspection does not "look back" over multiple lines which seems to be the only case I thought to use this inspection for.
